### ISSUE TITLE
Incompatible cucumber versions?

### ISSUE THREAD

**Issue by BevanR** [Adezandee/cucumber-mink#169]
Steps to reproduce;

1. `npm install -g cucumberjs`
1. Copy `test/features/` from this repository to `features/` in a new project/directory
1. `npm install --save cucumber-mink`
1. `cucumberjs` fails with:

```
> cucumberjs

/usr/local/lib/node_modules/cucumber/lib/cucumber/runtime/event_broadcaster.js:30
            throw error;
            ^
../../node_modules/cucumber-mink/dist/mink.js:261 function accepts a callback and returns a promise
```

cucumber.js version 1.2.2
cucumber-mink version 1.4.1

**Comment by BevanR** [comment]
This is caused by a function that Cucumber JS detects to both return a promise and accept a "done" callback.  Cucumber JS fails in this instance to avoid the step definition causing two Cucumber threads to run.  This can happen if the step definition function resolves the promise _and_ calls the "done" callback.  Cucumber JS' code to detect this scenario is not perfect; for example, I don't think it can't tell if the return is implicit, such as with an ES6 arrow function or from an explicit `return new Promise...` statement (or equivalent).

So the error might be a false negative, or not.

**Comment by Adezandee** [comment]
Fixed with #199, thanks for your report ! 👍
